jinahub/rankers/LightGBMRanker/README.md

```diff
@@ -0,0 +1,99 @@
+# ✨ LightGBMRanker
+
+**LightGBMRanker** is a Jina ranker, using the [LightGBM](https://github.com/microsoft/LightGBM) library, mode specifically, the `LightGBMRanker` is used for learning-to-rank.
+
+`LightGBMRanker` retrieves `query_features`, `match_features` and `relevance_label` stored inside `Document` object from `DocumentArray`, and builds a feature-label dataset to train the model.
+
+**Table of Contents**
+
+- [🌱 Prerequisites](#-prerequisites)
+- [🚀 Usages](#-usages)
+- [🎉️ Example](#%EF%B8%8F-example)
+- [🔍️ Reference](#%EF%B8%8F-reference)
+
+
+## 🌱 Prerequisites
+
+- This Executor works on Python 3.7 and 3.8. 
+- Make sure to install the [requirements](requirements.txt)
+- Refer to LightGBM [documentation](https://github.com/microsoft/LightGBM/tree/master/examples/lambdarank) to learn how to use LightGBM to train a ranker.
+
+## 🚀 Usages
+
+### 🚚 Via JinaHub
+
+#### using docker images
+
+Use the prebuilt images from JinaHub in your Python code: 
+
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub+docker://LightGBMRanker')
+```
+
+or in the `.yml` config.
+	
+```yaml
+jtype: Flow
+pods:
+  - name: ranker
+    uses: 'jinahub+docker://LightGBMRanker'
+```
+
+#### using source code
+Use the source code from JinaHub in your Python code:
+
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub://LightGBMRanker')
+```
+
+or in the `.yml` config.
+
+```yaml
+jtype: Flow
+pods:
+  - name: ranker
+    uses: 'jinahub://LightGBMRanker'
+```
+
+
+	
+
+## 🎉️ Example 
+
+```python
+from jina import Flow, DocumentArray
+
+f = Flow().add(uses='jinahub://LightGBMRanker')
```

Comment:
Won't this example crash? You are not providing required arguments in the init function